src/test/javascript/spec/component/account/vcs-access-token-settings.component.spec.ts

```diff
@@ -0,0 +1,91 @@
+    let comp: VcsAccessTokensSettingsComponent;
+
+    let accountServiceMock: { getAuthenticationState: jest.Mock; deleteUserVcsAccessToken: jest.Mock; addNewVcsAccessToken: jest.Mock };
+    let profileServiceMock: { getProfileInfo: jest.Mock };
+    let translateService: TranslateService;
+
+    const token = 'initial-token';
+
+    beforeEach(async () => {
+        profileServiceMock = {
+            getProfileInfo: jest.fn(),
+        };
+        accountServiceMock = {
+            getAuthenticationState: jest.fn(),
+            deleteUserVcsAccessToken: jest.fn(),
+            addNewVcsAccessToken: jest.fn(),
+        };
+
+        await TestBed.configureTestingModule({
+            imports: [ArtemisTestModule],
+            declarations: [VcsAccessTokensSettingsComponent, TranslatePipeMock],
+            providers: [
+                { provide: AccountService, useValue: accountServiceMock },
+                { provide: ProfileService, useValue: profileServiceMock },
+                { provide: TranslateService, useClass: MockTranslateService },
+                { provide: NgbModal, useClass: MockNgbModalService },
+            ],
+        }).compileComponents();
+        fixture = TestBed.createComponent(VcsAccessTokensSettingsComponent);
+        comp = fixture.componentInstance;
+        translateService = TestBed.inject(TranslateService);
+        translateService.currentLang = 'en';
+    });
+
+    beforeEach(() => {
+        profileServiceMock.getProfileInfo.mockReturnValue(of({ activeProfiles: [PROFILE_LOCALVC] }));
+        accountServiceMock.getAuthenticationState.mockReturnValue(of({ id: 1, vcsAccessToken: token, vcsAccessTokenExpiryDate: '11:20' } as User));
+        accountServiceMock.addNewVcsAccessToken.mockReturnValue(of({ id: 1, vcsAccessToken: token, vcsAccessTokenExpiryDate: '11:20' } as User));
+        accountServiceMock.deleteUserVcsAccessToken.mockReturnValue(of({}));
+    });
+
+    it('should initialize with localVC profile', async () => {
+        comp.ngOnInit();
+        expect(profileServiceMock.getProfileInfo).toHaveBeenCalled();
+        expect(accountServiceMock.getAuthenticationState).toHaveBeenCalled();
+        expect(comp.localVCEnabled).toBeTrue();
+    });
+
+    it('should initialize with no localVC profile set', async () => {
+        profileServiceMock.getProfileInfo.mockReturnValue(of({ activeProfiles: [] }));
+        comp.ngOnInit();
+        expect(profileServiceMock.getProfileInfo).toHaveBeenCalled();
+        expect(accountServiceMock.getAuthenticationState).toHaveBeenCalled();
+        expect(comp.localVCEnabled).toBeFalse();
+    });
+
+    it('should create new vcs access token', () => {
+        const newToken = 'new-token';
+        accountServiceMock.addNewVcsAccessToken.mockReturnValue(of({ id: 1, vcsAccessToken: newToken, vcsAccessTokenExpiryDate: '11:20' } as User));
+        comp.ngOnInit();
+        expect(comp.currentUser.vcsAccessToken).toEqual(token);
```

Comment:
The build is failing for the client tests. Since we know that the current user should be defined, we can use

```suggestion
        expect(comp.currentUser!.vcsAccessToken).toEqual(token);
```

to fix the build error